suite-native/navigation/src/types.tsx

```diff
@@ -37,6 +37,12 @@ export type StackToStackCompositeNavigationProps<
 
 export type RouteProps<T extends ParamListBase, K extends keyof T> = RouteProp<T, K>;
 
-export interface TabsOptions {
-    [tabName: string]: { iconName: IconName; label?: string; isActionTabItem?: boolean };
-}
+export type TabsOptions = {
+    [routeName: string]: {
+        routeName: string;
+        iconName: IconName;
+        label?: string;
+        isActionTabItem?: boolean;
+        params?: object;
```

Comment:
I would prefer `Record<string, unknown>` because I think `object` could be anything in JS even functions are objects I think